### PR TITLE
Further i18n work

### PR DESCRIPTION
**Pull request by raphaelm**
Still missing:

- [ ] proper string interpolation (e.g. gravatar description with link, user joined, etc)
- [ ] moment.js locale

### CODE REVIEW

File: webapp/src/locales/en.js [hunk 1]
```diff
@@ -1,6 +1,36 @@
...
+	'profileprompt:gravatar-email-hint': 'your address is not being sent to any server',
+	'profileprompt:gravatar-connect': 'Connect',
+	'profileprompt:submit-new': 'join',
+	'profileprompt:submit-existing': 'update',
 	About: 'About',
 	Schedule: 'Schedule',
-	Administration: 'Administration'
+	Administration: 'Administration',
+	Stages: 'Stages',
+	Channels: 'Kanäle'
```
Comment (rashfael) [inline webapp/src/locales/en.js:35]: Wrong language?

**Comment by rashfael** [comment]
I'd like it more if we'd had some sort of system for the keys. Right now the keys look very unfriendly against future changes.
For example `createchat:placeholder` is not a placeholder for the whole prompt, something like `CreateChatPrompt:input#name:placeholder` would be clearer, even if it seems exessive right now. I'd also tend to use the real file names of the components instead of another name for them (including capitalization).
Opinions?

**Comment by raphaelm** [comment]
>  For example createchat:placeholder is not a placeholder for the whole prompt, something like CreateChatPrompt:input#name:placeholder would be clearer, even if it seems exessive right now. 

I agree, and I also didnt feel sure about this. However, ``CreateChatPrompt:input#name:placeholder`` feels very technical and probably pretty repetitive. I already stumbled over this when adding the "create" button twice, do we really need translators to translate "create" twice? 

That said, I've never been happy with how this is handled on other platforms (english as base strings in django, weird identifiers without any convention on android), so it's not like I have a better option in mind…

> I'd also tend to use the real file names of the components instead of another name for them (including capitalization).
Opinions?

Agreed.

**Comment by rashfael** [comment]
> I already stumbled over this when adding the "create" button twice, do we really need translators to translate "create" twice?

I also thought about this, we can probably do a "generic" namespace.

**Comment by raphaelm** [comment]
Okay, I changed it for a ``File:element:type`` scheme, where the ``File`` is the ``.vue`` filename, and ``type`` is what the text is in relation to the element (text, label, placeholder, …). I took a little more free-form approach to ``element`` and ``type`` than actual CSS-like element selectors, since these seem too technical and instable. The translator doesn't care it it's a button or a link, if it's tecnically an input or a select, so I tried to better capture the meaning of the string there. We can still go for more specific selectors if we have very complex components.

**Pull request merged**